src/components/FeaturedSec.tsx

```
@@ -0,0 +1,69 @@
+import { cn } from "@/lib/utils";
```

ESLint

[error] 'cn' is defined but never used.

(@typescript-eslint/no-unused-vars)